xskillscore/core/probabilistic.py

```diff
@@ -207,7 +251,14 @@ def crps_ensemble(
         return res.mean(dim, keep_attrs=keep_attrs)
 
 
-def brier_score(observations, forecasts, dim=None, weights=None, keep_attrs=False):
+def brier_score(
+    observations,
+    forecasts,
+    dim=None,
+    weights=None,
+    keep_attrs=False,
+    member_dim="member",
```

Comment:
I would think this should be `None`. By default this will try to take mean over `member` which might not exist. This is just what we call it in `climpred`, but many people will use `ensemble`, `M`, etc.

Reply:
there is a check whether member_dim exists. if people want to use a different member_dim name they can specify.

Reply:
the expects not to find a member_dim. maybe I just delete that